content/docs/command-reference/fetch.md

```diff
@@ -73,43 +71,43 @@ or `-T` options are used).
 
 ## Options
 
-- `-r <name>`, `--remote <name>` - name of the
-  [remote storage](/doc/command-reference/remote) to fetch from (see
-  `dvc remote list`).
+- `-r <name>`, `--remote <name>`- name of the
+  [remote storage](/doc/command-reference/remote) to fetch from
+  (see`dvc remote list`).
```

Comment:
Unrelated changes? :slightly_frowning_face:

Reply:
Ah, I suppose it is a linter doing this?

Reply:
That's weird. Fixed in b01ccd95 too.